parsing/lexer.mll

```diff
@@ -316,6 +328,8 @@ let dotsymbolchar =
 let kwdopchar =
   ['$' '&' '*' '+' '-' '/' '<' '=' '>' '@' '^' '|']
 
+let extattrident = identchar+ ('.' identchar+)*
```

Comment:
This would require a comment to explain why the definition is as such. If I understand correctly, you are trying to accept the same as the parser's `attr_id` category, modulo whitespace differences.

The parser definition requires components (between the dots) to be valid OCaml identifiers or keywords. Your definition accepts components that start with a number. Why not require a `(lowercase | uppercase)` at the head of components?

Reply:
Done.